### PR TITLE
add car entity

### PR DESCRIPTION
**Pull request by levinbuilds**

### CODE REVIEW

**Review by Xittus — COMMENTED** [review]
where are the tests n?

**Review by Xittus — APPROVED** [review]

File: src/main/java/dev/bbzblit/m426/controller/CarController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Validated
+public class CarController {
+    public final CarService service;
+
+    public CarController(CarService carService){
```
Comment (Yanni8) [inline src/main/java/dev/bbzblit/m426/controller/CarController.java]: variable should be `final`

File: src/main/java/dev/bbzblit/m426/controller/CarController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @GetMapping("/api/car/{id}")
+    public ResponseEntity<Car> one(@PathVariable Long id) {
+        Car car = service.getCar(id);
+        return new ResponseEntity<>(car, HttpStatus.OK);
+    }
+
+    @PostMapping("/api/car")
+    public ResponseEntity<Car> insert(Car car) {
```
Comment (Yanni8) [inline src/main/java/dev/bbzblit/m426/controller/CarController.java]: Car variable is missing `@RequestBody` (not only on this endpoint)

File: src/main/java/dev/bbzblit/m426/service/CarService.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+public class CarService {
+    private final CarRepository repository;
+
+    public CarService(CarRepository carRepository){
```
Comment (Yanni8) [inline src/main/java/dev/bbzblit/m426/service/CarService.java]: variable should be final

**Review by Xittus — APPROVED** [review]

**Review by Yanni8 — APPROVED** [review]

**Pull request merged**